
This patch is not required for build, because freebsd-glue provides
<nlist.h>, however it is still required when using kvm.h in an external
package. See #729536.

--- a/lib/libkvm/kvm.h
+++ b/lib/libkvm/kvm.h
@@ -35,7 +35,7 @@
 
 #include <sys/cdefs.h>
 #include <sys/types.h>
-#include <nlist.h>
+#include <bsd/nlist.h>
 
 /* Default version symbol. */
 #define	VRS_SYM		"_version"
